homework34/src/main/java/hw34/CartController.java

```diff
@@ -0,0 +1,89 @@
+package hw34;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Scanner;
+
+@Component
+@Scope("prototype")
+public class CartController {
```

Comment:
The class naming is very, very debatable :) Usually, pattern for classes `*Controller` is applied for real HTTP controllers.